.nvmrc

```diff
@@ -0,0 +1 @@
+16
```

Comment:
I don't expect any problem, but could you check if node 18 also works fine (node 16 will go EOL in +- 7 months from now)

Reply:
Preferably in both places, or maybe even better, can `.nvmrc` be removed such that you are forced to specify it?

Reply:
I'm not sure if I follow - how removing the `.nvmrc` file is better? `.nvmrc`'s purpose is to specify that version for you, similar to how `.editorconfig`'s file is to specify IDE's / editor's configuration.